.gitignore

```diff
@@ -26,3 +26,4 @@ src/Backend/weblibdedx.js
 .gitignore
 public/weblibdedx.wasm
 .vscode/settings.json
+public/weblibdedx.data
```

Comment:
Do we need `public/weblibdedx.data` for correct execution of the GUI ?
If yes, then from where can we get it ?
Can we add it to github actions artifacts, just like webassembly products, i.e:
![image](https://user-images.githubusercontent.com/7374733/143618797-0f9ac306-d2b5-42bb-8b91-c7ab48d8babc.png)

Reply:
Oh, forgot about the workflow - It's from the same place the weblibdedx.wasm is comming from.

Reply:
Turns out it's not needed - everything is already compiled in the .js file. It isn't the most efficient setup but the --preload doesn't seem for work with react to well. It shouldn't be that bad as the overall size of the data is ~~3MB counting with spare .h and .py files which will be removed from that folder one day

Reply:
if we don't need that, then remove it from `.gitignore`